packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -5,6 +5,7 @@
 import '@webex/internal-plugin-mercury';
 import {forEach} from 'lodash';
 import {WebexPlugin} from '@webex/webex-core';
+import 'webrtc-adapter';
```

Comment:
What is the purpose of this import?

Reply:
This is required in order for reachability to work on Safari. WebRTC adapter fixes and creates a base line for all browsers. This includes it within webex.js file. It's listed here because it's related to meetings-plugin.